core/app/models/spree/taxon.rb

```diff
@@ -25,15 +25,27 @@ class Taxon < Spree::Base
     after_save :touch_ancestors_and_taxonomy
     after_touch :touch_ancestors_and_taxonomy
 
-    has_attached_file :icon,
-      styles: { mini: '32x32>', normal: '128x128>' },
-      default_style: :mini,
-      url: '/spree/taxons/:id/:style/:basename.:extension',
-      path: ':rails_root/public/spree/taxons/:id/:style/:basename.:extension',
-      default_url: '/assets/default_taxon.png'
-
-    validates_attachment :icon,
-      content_type: { content_type: ["image/jpg", "image/jpeg", "image/png", "image/gif"] }
+    if ::Spree::Config.taxon_attachment_module.blank?
+      Spree::Deprecation.warn <<-MESSAGE.strip_heredoc + "\n\n"
+        Using Paperclip as taxon_attachment_module for Taxon.
+
+        Please configure Spree::Config.taxon_attachment_module in your store
+        initializer.
+
+        To use the ActiveStorage adapter (recommended):
+          Spree.config do |config|
+            config.image_attachment_module = 'Spree::Taxon::ActiveStorageAttachment'
```

Comment:
this should be `config.taxon_attachement_module`

Reply:
thanks good catch!

Reply:
is this still unsolved?